public/main.js

```diff
@@ -166,41 +166,71 @@ function addEntryToCache(category, entry) {
   cachedData[category].sort(compareCachedObjects);
 }
 
-// Displays a data in the UI.
-function displayEntry(id, timestamp, category, type, note) {
-  var div = document.getElementById(id) || createAndInsertEntry(category, id, timestamp);
-
-  let time = new Date(timestamp.toMillis());
+function displayDatetimeString(milliseconds) {
+  let time = new Date(milliseconds);
   let dateString = (time.getMonth() + 1) + "/" + time.getDate() + "/" + time.getFullYear();
   let hours = time.getHours();
   let hoursString = hours == 0 ? 12 : (hours > 12 ? hours - 12 : hours);
   let timeString = hoursString + ":" + (time.getMinutes() < 10 ? "0" + time.getMinutes() : time.getMinutes()) + 
-  		" " + (hours >= 12 ? "PM" : "AM");
-  let categoryString = '';
+      " " + (hours >= 12 ? "PM" : "AM");
+  return dateString + " " + timeString;
+}
+
+function displayTypeString(category, type) {
```

Comment:
This should be "calculateTypeString" or "getTypeString"? The word "display" is confusing because it's not being displayed in this function.